src/types/Filter.ts

```diff
@@ -0,0 +1,5 @@
+export enum Filter {
+  ALL,
+  ACTIVE,
+  COMPLETED,
+}
```

Comment:
enum's elments should be in CamelCase

Reply:
Fix it 
![image](https://github.com/mate-academy/react_todo-app/assets/97547824/84558245-9ccd-4588-a75e-ca2bd3954124)